jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/FileSystemPool.java

```diff
@@ -62,12 +71,16 @@ public Resource.Mount mount(URI uri) throws IOException
         {
             try
             {
-                fileSystem = FileSystems.newFileSystem(uri, Collections.emptyMap());
+                fileSystem = FileSystems.newFileSystem(uri, ENV_MULTIRELEASE_RUNTIME);
             }
             catch (FileSystemAlreadyExistsException fsaee)
             {
                 fileSystem = Paths.get(uri).getFileSystem();
             }
+            catch (ProviderNotFoundException pnfe)
```

Comment:
Why do you want to move on if such exception is thrown? If there's no FS provider, we should not try to retain that filesystem and return a broken mount.

Reply:
Not an IAE